prow/gerrit/reporter/reporter.go

```diff
@@ -82,19 +82,20 @@ func (c *Client) ShouldReport(pj *v1.ProwJob) bool {
 
 	// Only report when all jobs of the same type on the same revision finished
 	selector := labels.Set{
-		client.GerritRevision: pj.ObjectMeta.Labels[client.GerritRevision],
-		kube.ProwJobTypeLabel: pj.ObjectMeta.Labels[kube.ProwJobTypeLabel],
+		client.GerritRevision:    pj.ObjectMeta.Labels[client.GerritRevision],
+		kube.ProwJobTypeLabel:    pj.ObjectMeta.Labels[kube.ProwJobTypeLabel],
+		client.GerritReportLabel: pj.ObjectMeta.Labels[client.GerritReportLabel],
```

Comment:
:joy: not all gerrit prowjobs has this label :joy: - we had a default LGTM label but that's not explicit!

Reply:
done

Reply:
(also here)